packages/person/src/person-search/element.ts

```diff
@@ -0,0 +1,303 @@
+ * @property {string} dropdownHeight Sets max-height of list so user can scroll trough results
+ * @property {string} graphic Icon to show before each fwc-list-item. If you want an icon only on one list-item then use the graphic property on the SearchableDropdownResultItem
+ * @property {string} initialText Text to display in dropdown before/without querystring in fwc-textinput
+ * @property {string} label Label for fwc-textinput element
+ * @property {string} leadingIcon Leading Icon to display in fwc-text-input
+ * @property {string} meta Icon to show after each fwc-list-item. If you want an icon only on one list-item then use the meta property on the SearchableDropdownResultItem
+ * @property {string} multiple Able to select multiple items
+ * @property {string} placeholder Placeholder text for fwc-textinput element
+ * @property {string} selectedId ID that should be highlighted in dropdown
+ * @property {string} value value for TextInput element
+ * @property {'page' | 'page-outlined' | 'page-dense' | 'header' | 'header-filled'} variant Set variant to header|page style
+ *
+ * @fires action Fires when a selection has been made on the fwc-list element
+ */
+export class PersonSearchElement
+  extends LitElement
+  implements SearchableDropdownProps, SearchableDropdownControllerHost
+{
+  // static shadowRootOptions = { ...Object.assign(LitElement.shadowRootOptions, { delegatesFocus: true }) };
+
+  /* style object css */
+  static styles = [sddStyles];
+
+  controller = new PersonSearchController(this);
+
+  /* Label passed to the fwc-text-input component */
+  @property()
+  label = '';
+
+  /* Placeholder passe to fwc-textinput */
+  @property()
+  placeholder = '';
+
+  /* Textinput variant passed to the fwc-text-input component */
+  @property()
+  variant = 'page';
+
+  /* The leading icon to display in fwc-textinput */
+  @property()
+  leadingIcon = 'search';
+
+  /* The trailing icon to display in fwc-textinput */
+  @property({ attribute: false, state: true })
+  trailingIcon = '';
+
+  /* The icon string to render in result list items on the meta slot */
+  @property()
+  meta = '';
+
+  /* value passed to textinput */
+  @property()
+  value = '';
+
+  /* The icon string to render in result list items on the graphic slot */
+  @property()
+  graphic = '';
+
+  /* The initial text in the dropdown before keyup event */
+  @property()
+  initialText = 'Start typing to search';
+
+  /* The leading icon to display in fwc-textinput */
+  @property()
+  multiple = false;
+
+  /* Label passed to the fwc-text-input component */
+  @property()
+  dropdownHeight = '250px';
+
+  @property()
+  disabled = false;
+
+  @property()
+  selectedId = '';
+
+  @property()
+  autofocus = false;
+
+  @query('fwc-textinput')
+  textInputElement: TextInputElement | undefined;
+
+  @query('fwc-list')
+  listElement: ListElement | undefined;
+
+  /* Build fwc-list-items */
+  protected buildListItem(item: SearchableDropdownResultItem): HTMLTemplateResult {
```

Comment:
this works... but i would prefer if you instead create a search-check-list-item and a search-list-item and kept the logic for rendering an item within that class